### ISSUE TITLE
Add option to suppress singleton warnings

### ISSUE THREAD

**Issue by maximilianh** [pybel/pybel#200]
What does this mean:

```
WARNING:pybel.parser:Added singleton [line 221582]: complex(p(HGNC:ESR1),g(HGNC:CDC42BPA)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221583]: complex(p(HGNC:ESR1),g(HGNC:CDK10)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221584]: complex(p(HGNC:ESR1),g(HGNC:CEP290)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221585]: complex(p(HGNC:ESR1),g(HGNC:CFDP1)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221586]: complex(p(HGNC:ESR1),g(HGNC:CHCHD3)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221587]: complex(p(HGNC:ESR1),g(HGNC:CHCHD4)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221588]: complex(p(HGNC:ESR1),g(HGNC:CHD1L)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221589]: complex(p(HGNC:ESR1),g(HGNC:CHMP4B)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221590]: complex(p(HGNC:ESR1),g(HGNC:CITED2)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221591]: complex(p(HGNC:ESR1),g(HGNC:CLASP2)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221592]: complex(p(HGNC:ESR1),g(HGNC:CLPB)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221593]: complex(p(HGNC:ESR1),g(HGNC:COL20A1)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221594]: complex(p(HGNC:ESR1),g(HGNC:COL4A3BP)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221595]: complex(p(HGNC:ESR1),g(HGNC:COPE)). Putative error - needs checking.
WARNING:pybel.parser:Added singleton [line 221596]: complex(p(HGNC:ESR1),g(HGNC:CORO2A)). Putative error - needs checking.
```

Is it normal that I'm getting tens of thousands of warning with the "official" large corpus file?

**Comment by cthoyt** [comment]
Because of the way the parser works, if there's an issue with the predicate or the object, it will try and just add the subject (since it lexes from left to right) so I've added in a warning message. Like it said, this just means it's worth checking.

The Selventa large corpus makes a lot of entries for the existence of complexes that aren't part of a relationship, so it makes these warnings. Maybe it's a good idea to add an option to suppress these (if you're sure that your BEL is coded properly)

**Comment by maximilianh** [comment]
OK, thanks!

I successfully ran through, I now have the CSV file. This is an awesome
converter!

**Comment by maximilianh** [comment]
You may want to upload the .cx result of the large corpus onto ndex. But
please also add a link to the .csv file, the .cx file format is terribly
hard to parse and understand. .csv is a lot better.
